Dockerfile

```diff
@@ -4,7 +4,13 @@ ARG KUBE_VERSION=1.12.0
 ENV HOME=/srv
 WORKDIR /srv
 
-RUN apk add --no-cache curl ca-certificates
+RUN apk -v --update add --no-cache \
+      curl ca-certificates \
+      python \
```

Comment:
Probably better to change to using the `python:3-alpine` base image, rather than adding python and pip separately.